new-site/docs-site/content/0.19.0/api/usage/create.md

```diff
@@ -0,0 +1,209 @@
+# Create a collection
```

Comment:
Could you nest all collection related operations under a single "Collections" section and then all document related operations under a single "Documents" section? 

So something like:

- Collections
  - Create a collection
  - Retrieve a collection
  - List all collections
  - Drop a collection
  - Etc
- Documents 
  - Index a document (this is missing btw)
  - Search for documents
  - Import documents
  - Etc

Reply:
Fine. Will work on that